roles/aks-apply/files/dev/otp-data-builder-finland-v3-street-graph-only-dev.yml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: otp-data-builder-finland-v3-street-graph-only-dev
+spec:
+  schedule: "0 1 * * *" # schedule in Finland time
+  timeZone: "Europe/Helsinki"
+  successfulJobsHistoryLimit: 4
+  failedJobsHistoryLimit: 4
+  jobTemplate:
+    spec:
+      backoffLimit: 2 # max two retries
+      template:
+        metadata:
+          labels:
+            app: otp-data-builder-finland-v3-street-graph-only-dev
+        spec:
+          activeDeadlineSeconds: 9000 # one build can take max 2 hours and 30 minutes
+          restartPolicy: OnFailure
+          nodeSelector:
+            agentpool: builderpool
+          tolerations:
+          - key: "builder"
+            operator: "Equal"
+            value: "yes"
+            effect: "NoSchedule"
+          containers:
+          - name: otp-data-builder-finland-v3-street-graph-only-dev
+            image: hsldevcom/otp-data-builder:split-build-test
+            volumeMounts:
+            - mountPath: /opt/otp-data-builder/storage
+              name: datastorage
+            imagePullPolicy: Always
+            securityContext:
+              privileged: true
+              capabilities:
+                add:
+                 - NET_ADMIN
+                 - SYS_ADMIN
+            env:
+            - name: ROUTER_NAME
+              value: "finland"
+            - name: BUILDER_TYPE
+              value: "Finland dev v3"
+            - name: DOCKER_AUTH
+              valueFrom:
+                secretKeyRef:
+                  name: docker-auth
+                  key: docker-auth
+            - name: DOCKER_TAG
+              value: "v3"
+            - name: DOCKER_USER
+              valueFrom:
+                secretKeyRef:
+                  name: docker-user
+                  key: docker-user
+            - name: SLACK_ACCESS_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: slack-access-token
+                  key: slack-access-token
+            - name: SLACK_CHANNEL_ID
+              valueFrom:
+                secretKeyRef:
+                  name: slack-ci-channel-id
+                  key: slack-ci-channel-id
+            - name: OTP_TAG
+              value: "v2"
+            - name: SEED_TAG
+              value: "v3"
+            - name: TOOLS_TAG
+              value: "v3"
+            - name: JAVA_OPTS
+              value: "-Xmx15G -Xms15G"
+            - name: EXTRA_SRC
+              value: "{\"digitraffic\": {\"url\": \"https://rata-beta.digitraffic.fi/api/v1/trains/gtfs-passenger.zip\"}}"
+            - name: EXTRA_UPDATERS
+              value: "{\"digitraffic-trip-updates\": {\"url\": \"https://rata-beta.digitraffic.fi/api/v1/trains/gtfs-rt-updates\"}}"
```

Comment:
These shouldn't be needed here. Maintenance would be more complicated if these are defined in both builder stages.

Reply:
I removed these